### DISCUSSION TITLE
Trakt showing statistics differences with Showly

### DISCUSSION THREAD

**Discussion by conradseba** [trakt/showly#729] [Q&A]
Hey! I've used Showly for years now, but I recently spotted a problem with my tracked shows and movies.
I don't understand why I'm seeing different numbers in Showly and Trakt if I've been synching everything since the beginning. Also, just one single sync should solve this issue, but it doesn't. The amount of shows and movies is also off. Can it be explained? And which is the correct one? Thanks!!

![611b5de3-05e6-499c-afed-32e7c9d6f44a](https://github.com/michaldrabik/showly-2.0/assets/10448765/9938e961-bf08-419c-8fee-8f95332f8fe3)
![c88f1bb4-e5bc-4763-aaa9-68f70d8e93f7](https://github.com/michaldrabik/showly-2.0/assets/10448765/b43b8505-9d66-42c7-8656-a261ef8b3dea)

**Comment by michaldrabik** [comment]
Could be that you have multiple plays registered in Trakt. Multiple plays are not supported by Showly as of now.

**Reply by conradseba** [reply]
I thought so at first, and that could explain the discrepancy in days and hours, but not the amount of episodes or movies.

**Reply by michaldrabik** [reply]
It could also be if you have items in Hidden section. The progress from hidden section in showly is counted into statistics while I see that Trakt does not.

You might check it as well.

**Reply by conradseba** [reply]
I played a bit unhiding and rehiding shows to see if it has any effect on the amount of watched episodes, but that's not it. Also, because I have no hidden movie, and there's the movie difference too.

**Reply by michaldrabik** [reply]
Do you maybe have anything listed in here? https://trakt.tv/settings/hidden

**Reply by conradseba** [reply]
this is where I unhided/rehided stuff. Made no difference ¯\_(ツ)_/¯

**Reply by michaldrabik** [reply]
Ok to validate this please unhide everything from Trakt.tv website and Showly app. See if the stats are matching then.

**Reply by conradseba** [reply]
uff, that's too much of a hassle man, I've tons of hidden/abandoned shows. Why don't we try with the movies? what do you suggest to do to tackle that one? There're 9 more movies in Showly than Trakt in my list.

**Reply by michaldrabik** [reply]
Yeah it's hard to check remotely and since my accounts are all matching. So you got more movies in My Movies section then Trakt has in history?

**Reply by conradseba** [reply]
![photo1693754697](https://github.com/michaldrabik/showly-2.0/assets/10448765/6b609ad4-9909-4b64-b922-2cb65a56a045)
![photo1693754697 (1)](https://github.com/michaldrabik/showly-2.0/assets/10448765/a6449450-c628-43af-be25-a3f153a1be84)

Yes! this is the difference today, shorter than before. Yet, the point is, why is there a difference at all?
Cheers!

**Reply by conradseba** [reply]
This is getting worse and worse for me, it just doesn't make any sense, so weird. Showly, Trakt and Cinexplore, which all 3 are supposed to be in sync, show different numbers. But let Cinesplore aside, Showly is still showing very numbers as Trakt. :(
![Screenshot_20240319-225702_Cinexplore](https://github.com/michaldrabik/showly-2.0/assets/10448765/7be9f6f0-4c56-4e73-91c6-ea2d14e48189)
![Screenshot_20240319-225737_Showly](https://github.com/michaldrabik/showly-2.0/assets/10448765/6fd2f46a-5840-4b20-8cf2-a52b9d498163)
![Screenshot_20240319-230425_Trakt](https://github.com/michaldrabik/showly-2.0/assets/10448765/d3d60a06-a6ef-4ed3-94e4-30cb0b29e973)
